.gitignore

```diff
@@ -9,3 +9,4 @@ out
 .idea
 openapi.yaml
 .env
+*.DS_Store
```

Comment:
Why the `*` prefix?

Reply:
Ah, okay, We shouldn't need that -- unqualified filenames should be matched in any directory at or under the level where the `.gitignore` file lives.

`*.DS_Store` would match `Foo.DS_Store` as a filename.

Reply:
Err, to be more helpful: this can be `.DS_Store` rather than `*.DS_Store`.